cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/Transformer.java

```diff
@@ -26,6 +27,8 @@
 
     String keyToTransform();
 
+    void setSensitivityLevel(SensitivityLevel sensitivityLevel);
```

Comment:
@gituser173 @Bonczidai I think it is better to do it using overload, it is more natural to keep it stateless.